docs/blog/es-tooling.md

```diff
@@ -0,0 +1,71 @@
+---
+title: ES Tooling - community lead e18e-focused tooling
+author:
+  - name: The e18e Contributors
```

Comment:
Perhaps it would be good to give credit to yourself 😄 I imagine we could have other guest writers in the future and it would be good to give the same credit too.

Reply:
Ya 💯
Maybe this one could be signed as James and Passle if he is also going to do one of the sections?